pkg/executor/agent/agent.go

```diff
@@ -95,9 +95,9 @@ func Run(ctx context.Context, r runner.Runner, args []string) {
 
 	if r.GetType().IsMain() {
 		output.PrintEvent("test execution finished", e.Id)
+		output.PrintResult(result)
 	}
 
-	output.PrintResult(result)
```

Comment:
please check, that it will not break checking for errors in init and scraper executors

Reply:
there is no error passed in this line - they are above.